internal/saucecloud/cypress.go

```diff
@@ -102,6 +102,10 @@ func (r *CypressRunner) runSuites(fileURI string) bool {
 	// Submit suites to work on.
 	go func() {
 		for _, s := range suites {
+			retries := r.Project.GetSauceCfg().Retries
+			if s.PassThreshold.MaxAttempts > 0 {
+				retries = 0
+			}
```

Comment:
Hm. This side effect is not very obvious and would have to be carefully documented. Rather than go this way...
why not combine the two to reduce complexity? Since I can already define retries, perhaps all we need to introduce is just _passThreshold_?

Reply:
we did several discussions about it. Setting `retries` is running once and appending `retries` times to it. The behavior is a bit different. 🤔

Reply:
> Setting retries is running once and appending retries times to it. The behavior is a bit different. 🤔

Currently, the `retries` behavior is the same as having a passThreshold with maxAttempts == retries and a threshold of 1, no? If the newly added behavior is different, where does it differ?

Reply:
hmm.. if `maxAttempt` is 4, the max attempt is 4. If `retries` is 4, the max attempt is 5. But I think it's fine to combine `maxAttempt` with `retries` and just add some explanation in the docs. Let me do the update